libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.spec.ts

```diff
@@ -193,13 +207,18 @@ describe('RadioGroupComponent', () => {
                 });
 
                 it('should update the selected radio when the bound field is updated', async () => {
-                  spectator.setHostInput('selected', dataScenario.items[2]);
+                  spectator.setHostInput('selected', dataScenario.items[0]);
                   // Wait for radio checked attribute to be updated;
-                  await TestHelper.whenTrue(() => radioChecked(2));
-
-                  expect(radioChecked(0)).toBeFalse();
-                  expect(radioChecked(1)).toBeFalse();
-                  expect(radioChecked(2)).toBeTrue();
+                  await TestHelper.whenTrue(() => radioChecked(0));
+
+                  expect(dataScenario.items.length).toEqual(ionRadioElements.length);
+                  ionRadioElements.forEach((_, index) => {
+                    if (index === 0) {
```

Comment:
Consider using a constant for the comparison similar to what you do in e.g. `'should have selected radio corresponding to the selected data item'` and `'should update the selected radio when the bound field is updated'`

Reply:
Changed 214 into:
```typescript 
expect(dataScenario.items).not.toHaveLength(0);```